src/rules/mod.rs

```diff
@@ -29,6 +29,7 @@ mod no_eval;
 mod no_ex_assign;
 mod no_explicit_any;
 mod no_func_assign;
+mod no_misused_new;
```

Comment:
can you add this rule in `fn get_all_rules` ?

```rust
no_misused_new::NoMisusedNew::new(),
```

Reply:
I forgot add that after fixing conflict my forked repo